lambdas/service/app.py

```diff
@@ -771,36 +806,76 @@ def get_order():
     return {'order': Plugin.load().service_config().order_config}
 
 
-@app.route('/manifest/files', methods=['GET'], cors=True)
+@app.route('/manifest/files', methods=['GET'], cors=True, spec={
+    'description': 'Initiate and check status of a manifest generation job, returning a either a 301 redirecting to a '
+                   'URL to re-check the status of the manifest generation or a 302 redirecting to the location of the '
+                   'completed manifest.',
+    'parameters': [
+        {
+            'in': 'query',
+            'name': 'filters',
+            'description': 'Filters to be applied when generating the manifest',
+            'required': False,
+            'schema': {
+                'type': 'string'
+            }
+        },
+        {
+            'in': 'query',
+            'name': 'format',
+            'description': 'The desired format of the output. Possible values are `compact` (the default) for a '
+                           'tab-separated manifest and `terra.bdbag` for a manifest in the format documented '
+                           '`http://bd2k.ini.usc.edu/tools/bdbag/. The latter is essentially a ZIP file containing two '
+                           'manifests: one for participants (aka Donors) and one for samples (aka specimens). The '
+                           'format of the manifests inside the BDBag is documented here: '
+                           'https://software.broadinstitute.org/firecloud/documentation/article?id=10954',
+            'required': False,
+            'schema': {
+                'type': 'string',
+                'enum': [
```

Comment:
Consider consolidating this list with https://github.com/DataBiosphere/azul/blob/225699a3af92a90d6bf1ad3653be2f0b52973b7a/lambdas/service/app.py#L898

Reply:
Also, not all of these are listen in the description, and there differences should be described.

Reply:
'tsv' and 'bdbag' are legacy values for 'compact' and 'terra.bdbag' respectively. I suspect we might want them mentioned in the description of the 'format' field, however probably don't want them listed as possible values.

Reply:
If we want to drive validation from the spec we need to mention the backward-compatible entries in the enum. The description of the `format` parameter should make them as deprecated.